docs/apache-airflow-providers-amazon/operators/transfer/sql_to_s3.rst

```diff
@@ -0,0 +1,50 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+SQL to Amazon S3 Transfer Operator
+==================================
+
+Use SqlToS3Operator to copy data from a SQL server to an Amazon Simple Storage Service (S3) file.
+SqlToS3Operator is compatible with any SQL connection as long as it is defined in Connections.
```

Comment:
This is not quite right. It support only hooks that implemented [`get_pandas_df` ](https://github.com/apache/airflow/blob/5276ef8ad9749b2aaf4878effda513ee378f4665/airflow/providers/amazon/aws/transfers/sql_to_s3.py#L161)

I'm not so though how to explain it in the doc. The users who read this might not know what `get_pandas_df` is. Maybe we should add something like "as long as the SQL hook has function that converts the sql result to pandas dataframe" (or something in that area)?